Taskfile.yml

```diff
@@ -207,3 +207,30 @@ tasks:
     cmds:
       - >-
         PYTHONPATH=$(pwd) poetry run python -W ignore utils/run_model.py {{.CLI_ARGS}}
+
+  update-requirements:
+    desc: Update the requirements.txt file for a pipeline script.
+    summary: |
+      Example usage:
+      task update-requirements -- pipeline/eval/requirements/eval.in
+    cmds:
+      # Make sure a file was given to update.
+      - >-
+        if [[ -z "{{.CLI_ARGS}}" ]]; then
+          echo "Provide a path to the .in file";
+          echo "For example:"
+          echo "task update-requirements -- pipeline/eval/requirements/eval.in";
```

Comment:
I assume the path should not be hardcoded here

Reply:
The path is just an example path.